.github/workflows/codeql.yml

```diff
@@ -1,7 +1,9 @@
 name: CodeQL Static Analysis
 on:
-  pull_request_target:
-    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+      - BIPS-*
```

Comment:
I'll ask Ian to review this. Just to make sure it's okay to use branches that start with BIPS, they wouldn't be the default branch but it might be okay because the issue was around pull_request_target